pkgs/desktops/gnome-3/3.24/core/gnome-settings-daemon/default.nix

```diff
@@ -11,11 +11,15 @@ stdenv.mkDerivation rec {
 
   buildInputs = with gnome3;
     [ intltool pkgconfig ibus gtk glib gsettings_desktop_schemas networkmanager
-      libnotify gnome_desktop lcms2 libXtst libxkbfile libpulseaudio
+      dconf libnotify gnome_desktop lcms2 libXtst libxkbfile libpulseaudio
       libcanberra_gtk3 upower colord libgweather xkeyboard_config
       polkit geocode_glib geoclue2 librsvg xf86_input_wacom udev libgudev libwacom libxslt
       libtool docbook_xsl docbook_xsl_ns wrapGAppsHook gnome_themes_standard ];
 
+  preFixup = with gnome3; ''
+    gappsWrapperArgs+=(--prefix GIO_EXTRA_MODULES : "${dconf}/lib/gio/modules")
```

Comment:
should this go as well? settings are broken for me without it but maybe I'm misunderstanding how it's meant to work?